website/install.md

```diff
@@ -21,14 +28,25 @@ Git Town runs anywhere Git and [Bash](https://www.gnu.org/software/bash/bash.htm
     </td>
     <td>
       <ul>
+        <li>On Arch Linux, install <code><a href="https://aur.archlinux.org/packages/git-town/">git-town</a></code> from the AUR</li>
```

Comment:
Maybe something along the lines of:
* Install `git-town` from the aur using your favorite helper
* Or, to install manually from the aur:
  * download the tarball from --link--, run `tar xvf /path/to/tarball`
  * run `cd git-town`
  * run `makepkg -si`